AWS/legos/aws_get_all_service_names/aws_get_all_service_names3.json

```diff
@@ -1,11 +1,11 @@
 {
-  "action_title": "AWS_Get_All_Service_Names_v2.1",
-  "action_description": "This retrieves a list of all AWS services in a region.",
+  "action_title": "AWS_Get_All_Service_Names_v3",
```

Comment:
Do we need the underscore? This may look odd while searching for a  lego (since other lego titles are without _'s) Just my opinion though.

Reply:
good catch. I'll fix